CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ haproxy Cookbook CHANGELOG
 ==========================
 This file is used to list changes made in each version of the haproxy cookbook.
 
+v1.9.7
+------
+- Lawrence: enable 1.5 admin features
```

Comment:
When we fork community cookbooks and create our own versions that match that of the upstream, I figure it saves a lot of pain when we mark a change as having come from us!